Add Cloud Service Certification Project Maintainer Role

mcleo-d opened the pull request:
## Description
This pull request adds the following **CSC Project Maintainer** role to the Cloud Service Certification `open-roles` folder ...

# CSC Project Maintainer - Cloud Service Certification 
Helping run [Cloud Service Certification](https://github.com/finos/cloud-service-certification) with the Cloud Service Certification Lead Maintainer as CSC Project Maintainer is a fun open source role involving a broad set of tasks that need to be taken care of, including ...

## Role Requirements

- You're an experienced senior level cloud services engineer within the financial services industry and understand the relationship between cloud engineering, finacial services regulation, cyber security, risk and banking policies within an enterprise banking setting.

- You'll review and provide feedback on pull requests. The CSC community is growing fast and we're receiving a growing number of pull requests. You will help review pull requests, provide constructive feedback and support contributors, and ultimately making the call on what is merged into the project.

- You'll help guide architectural decisions. The future of Cloud Service Certification is discussed and planned on project calls and in our GitHub Issues. You’ll help drive these conversations and will guide CSC towards the best possible solutions.

- You'll convert high-level ideas into detailed specifications and actionable issues. You will provide the required context to allow community contributors to quickly and effectively pick up issues.

- You'll educate and drive the quality of Cloud Service Certification deliverables and documentation. You'll work to identify and implement opportunities for getting the FINOS Community involed in contributing solutions to the project.

- You'll help define and uphold FINOS Cloud Service Certification community standards. You will represent Cloud Service Certification within the FINOS community, both with the quality of your solutions and the way you treat people in the community.

## Skills Requirements

The Cloud Service Certification Project Maintainer should ... 

- Be curious and like using critical thinking skills.
- Pride yourself on doing the best work you can do. 
- Enjoy helping other people.
- Be proactive and ask questions.
- Be flexible and responsive.
 
The Cloud Service Certification Project Maintainer will have to understand the [FINOS Landscape](https://landscape.finos.org), [Open Developer Platform](https://github.com/finos/open-developer-platform) and [FINOS Governance and Policies](https://github.com/finos/community/tree/master/governance). 

The Cloud Service Certification Project Maintainer will reach out to FINOS for assistance and support as they need it.

## Contact Cloud Service Certification

Contact [James McLeod](james@finos.org), FINOS Director of Community, or  [Peter Thomas](peter.thomas@db.com), Cloud Service Certification Lead Maintainer from Deutsche Bank, for more information or drop into the next [Cloud Service Certification Project Meeting](https://github.com/finos/cloud-service-certification/issues?q=label%3Ameeting+) to say hello.

🌤 🚀 🤖

mcleo-d commented on docs/open-roles/cloud-service-certification-project-maintainer.md:
```diff
@@ -0,0 +1,36 @@
+
+## Role Requirements
+
+- You're an experienced senior level cloud services engineer within the financial services industry and understand the relationship between cloud engineering, finacial services regulation, cyber security, risk and banking policies within an enterprise banking setting.
```
word `financial services regulation` slightly differently, to `ideally with`.

mcleo-d replied:
@peter-thomas-db

This is addressed in the commit below below ...

- update engineering experience to ideally within FS industry 5f5c4d1

James.

peter-thomas-db commented on docs/open-roles/cloud-service-certification-project-maintainer.md:
```diff
@@ -0,0 +1,36 @@
+
+- You'll review and provide feedback on pull requests. The CSC community is growing fast and we're receiving a growing number of pull requests. You will help review pull requests, provide constructive feedback and support contributors, and ultimately making the call on what is merged into the project.
+
+- You'll help guide architectural decisions. The future of Cloud Service Certification is discussed and planned on project calls and in our GitHub Issues. You’ll help drive these conversations and will guide CSC towards the best possible solutions.
```
Could we change this to 'You'll help to guide overall direction, setting the priority and strategy for the backlog and owning the overall direction of the project' for the first sentence.

peter-thomas-db commented on docs/open-roles/cloud-service-certification-project-maintainer.md:
```diff
@@ -0,0 +1,36 @@
+
+- You'll help guide architectural decisions. The future of Cloud Service Certification is discussed and planned on project calls and in our GitHub Issues. You’ll help drive these conversations and will guide CSC towards the best possible solutions.
+
+- You'll convert high-level ideas into detailed specifications and actionable issues. You will provide the required context to allow community contributors to quickly and effectively pick up issues.
```
Given that we are asking the sub teams to do most of this now, can we change this wording?  Perhaps to something like 'You will contribute to the detailed solutions and support the rest of the community in building out the features.'

mcleo-d replied:
@peter-thomas-db

This is addressed in the commit below below ...

- update @peter-thomas-db role feedback d86df20

James.

mcleo-d commented:
Hi @peter-thomas-db 

I have updated the Project Maintainer Role as advised in the commits below. I have also added commit links to the threads of your inline comments.

- update engineering experience to ideally within FS industry 5f5c4d1
- update @peter-thomas-db role feedback cae4272
- update @peter-thomas-db role feedback d86df20

Please feel free to feedback.

James.

abdullahgarcia commented:
@mcleo-d everything looks great!

Just one minor thing, can we make sure to remove ".DS_Store" files.

You can add these files into your ignore list so it works automatically.

abdullahgarcia reviewed (changes requested):
As stated before, all good, just pending to remove ".DS_Store" files.

abdullahgarcia reviewed (changes requested):
Hi James,

The change should be ideally in your local .gitignore. I think this change is fine for now, but it hasn't removed the .DS_Store files from the pull request.

mcleo-d commented:
> Hi James,
> 
> The change should be ideally in your local .gitignore. I think this change is fine for now, but it hasn't removed the .DS_Store files from the pull request.

Hi @abdullahgarcia,

The complete update and push is now ready for review. I was pulled away from the entire change before being able to push the final commits to the repo. 

I have also updated and pushed `.gitignore` to the repo to avoid accidental pushes from other CSC project contributors. Please see below.

1. add global .DS_Store to .gitignore 125615d
2. remove .DS_Store commits 569fd89
3. remove aws specific .DS_Store 855eba2
4. add newline 7169ada

All the best!

James.

mcleo-d commented:
@peter-thomas-db and @abdullahgarcia 

I have merged this PR based on @abdullahgarcia's feedback being completed here ... https://github.com/finos/cloud-service-certification/pull/103#issuecomment-795464459

Kind regards,

James.
